.devcontainer/vmr-source-build/build.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
```

Comment:
The name of this file is confusing, I thought for a minute that it actually builds the VMR, but it just runs the sync process. Maybe we could name it something else like `run-vmr-sync.sh`?

Reply:
But it does not that sync.  It also preps.  Maybe `prep.sh`

Reply:
`init.sh`?

Reply:
> `init.sh`?

I can get onboard with that.